src/main/java/seedu/duke/TimeTable.java

```diff
@@ -3,59 +3,70 @@
 import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.Calendar;
-import java.util.Locale;
+import java.util.NoSuchElementException;
 import java.util.Scanner;
```

Comment:
The 'delete a module' feature has been explained in 2 lines. 
Should you explain more in-depth?
Some things that could be further explained are the:
- data/variables that are used in the interaction 
- a diagram to illustrate the interaction (e.g a sequence diagram, UML diagram)
- the expected outputs and any changes in any states of the program 

![image](https://user-images.githubusercontent.com/60385689/97514986-fafe7380-19ca-11eb-8551-e34f6df248da.png)